apps/web/src/Providers.tsx

```diff
@@ -28,7 +28,7 @@ const queryClient = new QueryClient({
 
 const Providers: React.FC<PropsWithChildren<{}>> = ({ children }) => {
   return (
-    <ThemeProvider>
+    <ThemeProvider shouldDisableGlobals>
```

Comment:
🗒 note (non-blocking): This is one place that I'm worried about visual regressions, but without it, I'm seeing styles breaking in web. This is because the emotion globals / reset are not wrapped in layers, and thus take precedent over all the novui styles